en/sql/schema/table_stmt.rst

```diff
@@ -727,6 +736,13 @@ If you specify REUSE_OID together with the collation of table, it can be placed
     *   OID reusable tables are supported only by CUBRID 2008 R2.2 or above, and backward compatibility is not ensured. That is, the database in which the OID reusable table is located cannot be accessed from a lower version database.
     *   OID reusable tables can be managed as partitioned tables and can be replicated.
 
+.. _dont-reuse-oid:
+
+DONT_REUSE_OID
+^^^^^^^^^^^^^^
+
+You can specify the **DONT_REUSE_OID** option when creating a table, then a referable table opposite to **REUSE_OID** is created.
```

Comment:
I don't understand the sentence easily, so I propose the change as below:

Specifying the **DONT_REUSE_OID** option when creating the table will create a referable table as opposite to **REUSE_OID**.

you can use 'referenceable' instead of 'referable'

Reply:
OK, i will apply your comment.

Reply:
```suggestion
Specifying the DONT_REUSE_OID option when creating the table will create a referable table as opposite to REUSE_OID.
```

Reply:
```suggestion
Specifying the **DONT_REUSE_OID** option when creating the table will create a referable table as opposite to **REUSE_OID**.
```